tests/unit_tests/query_handler/test_scope_query_handler_context.py

```diff
@@ -284,18 +286,52 @@ def test_release_parent_before_child_with_temporary_object_expect_exception(
     parent = scope_query_handler_context
```

Comment:
I am assuming that a fresh context is injected here for each test, right?

Reply:
yes, this is the magic of pytest fixtures

Reply:
Okay.